tracer/src/Datadog.Trace.SourceGenerators/EnumExtensions/EnumExtensionsGenerator.cs

```diff
@@ -0,0 +1,140 @@
+﻿// <copyright file="EnumExtensionsGenerator.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Datadog.Trace.SourceGenerators.EnumExtensions.Diagnostics;
+using Datadog.Trace.SourceGenerators.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Datadog.Trace.SourceGenerators.EnumExtensions;
+
+/// <inheritdoc />
+[Generator]
+public class EnumExtensionsGenerator : IIncrementalGenerator
```

Comment:
>I thought your blog post for this was very insightful for the IIncrementalGenerator stuff.

Thanks! 🙂 

> Maybe worth tossing a link to it in a comment in this file? 

I thought about it, but I am writing a new couple of posts about the new APIs I'm using here (`ForAttributeWithMetadataName`) so it might make sense to add a link to those once they're written 🙂 

Yes, I just don't want to trigger CI again to add a comment when I have green across the board 😝

Reply:
Sounds good to me!